index.html

```diff
@@ -1,183 +1,287 @@
 <!DOCTYPE html>
```

Comment:
Your project is missing [linter setup files](https://github.com/microverseinc/linters-config/tree/master/html-css).
- Kindly add to your repo `
.hintrc ` and `.stylelintrc.json`.

- After your installation add your `node_modules` to your `.gitignore`.

Reply:
Thanks for the review, the second change had already been made previously.
![Screenshot from 2021-12-01 10-54-00](https://user-images.githubusercontent.com/91301423/144267720-241d592e-eef9-429c-a85e-6377cbe94746.png)